sky/packages/sky_engine/LICENSE

```diff
@@ -2473,6 +2473,36 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 --------------------------------------------------------------------------------
 dart
 
+Copyright (c) 2017 the Dart project authors.  Please see the AUTHORS file
```

Comment:
This implies that there's a new file in Dart that has the same copyright but without a comma. Can we add the comma back in so that it doesn't have a unique license?